dune

```diff
@@ -25,11 +114,18 @@
  (deps (:source stdcompat__native.ml_native))
  (action (run cp %{source} %{targets})))
 
+(rule
+ (targets stdcompat__init.ml.in)
+ (deps (:source stdcompat__init.mli.in))
+ (action (run cp %{source} %{targets})))
```

Comment:
I think you could use `copy_file` here, cp might not be available on the user's system.

Reply:
Yup, `copy` did the trick. Not sure why I used `cp` before.

Reply:
Ah, the `cp` was there upstream and I just didn't change it.